frontend/src/view/views/Maps.vue

```diff
@@ -72,34 +76,46 @@ export default {
       this.loading = true
       try {
         await this.getMaps()
+        this.orderMapIds()
```

Comment:
du sortierst hier die maps einmal, was ok ist :) BESSER ist es imo, wenn orderedMapIds ein computed property ist statt in data vorliegt :) dann ist es immer aktuell wenn this.maps sich aendert, und es faellt weniger visual overhead hier in dem file an, da nur ein computed benoetigt wird statt data und function und function call. die computed properties werden von vue gecacht. sobald sich eine abhaengigkeit aendert, aendert sich auch der computed wert. also falls sich die maps aendern, aendert sich auch orderedMapIds automatisch mit vue magic. https://vuejs.org/v2/guide/computed.html#Computed-Properties

vorgehen waere:
1. die zeile hier entfernen
2. bei data `orderedMapIds` entfernen
3. bei computed orderedMapIds hinzufuegen mit inhalt:
```
return Object.keys(this.maps).sort((a, b) => {
  const mapA = this.maps[a]
  const mapB = this.maps[b]
  if (mapA.weight === mapB.weight) {
    return 0
  }
  return mapA.weight < mapB.weight ? 1 : -1
})
```
4. orderMapIds function entfernen :)

Reply:
Das hab ich in der Tat fast ne Stunde lang probiert. Hat auch funktioniert, außer wenn ich direkt die Maps Seite geladen habe, dann hat das computed gegriffen bevor die Maps geladen wurden und die Seite war einfach leer.

Reply:
oki doki, geht ja so auch =) vll was fuer spaeter :)